### PR TITLE
fix: don't check if submission_uuid from bulk_fetch_workflow matches

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** For team submissions, we request assessment details using the team submission uuid. We look up the TeamStaffWorkflow for the team submission uuid, and then get the assessment associated with this workflow. The problem here is that that assessment is for one of the individual submissions, so the submission uuid on the assessment won't be the team submission uuid. 

This is a fix so that grading can happen, but we will need a more robust solution, since the grade not showing up on the listing page is this exact same issue, but without this workaround.

JIRA: [AU-654](https://2u-internal.atlassian.net/browse/AU-654)

**What changed?**

- Removed the check for non-matching submission uuids, since for now this is expected behavior.

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

- You should be able to grade team oras, override the grade for team ORAs, and view graded team oras locally, once you have short circuted the code that forces team ORAs to use demo mode.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by nsprenkle** [comment]
Where was this previously failing? The Staff Grading app?

**Comment by nsprenkle** [comment]
Some additional testing notes for folks who want to manually verify. You have to change this line to allow grading for teams in ORA Staff Grader:

https://github.com/openedx/edx-platform/blob/3b93d1428c2b0c7e0211af1f53391ab952ebf1b5/lms/djangoapps/ora_staff_grader/serializers.py#L186

Change to remove the teams disable:

```
        return obj['isEnabled'] # and not obj['oraMetadata'].teams_enabled
```

**Review by nsprenkle — APPROVED** [review]
Works as expected :shipit:

**Review by mattcarter — APPROVED** [review]
LGTM
